docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/hpc-launching-architecture.rst

```diff
@@ -0,0 +1,370 @@
+configuration options.
+
+-  ``slot_type``: The default slot type (``cuda``, ``rocm``, ``cpu``) assumed when users request
+   resources from Determined in terms of ``slots_per_trial``. May be overridden per
+   partition/resource pool.
+
+-  ``tres_supported``: Indicates if ``SelectType=select/cons_tres`` is set in the Slurm
+   configuration. It affects how Determined requests GPUs from Slurm. The default is true, which
+   allows an experiment to request GPU ``slots_per_trial`` without having to specify
+   ``slurm.slots_per_node``, and Slurm will automatically allocate the number of nodes needed to
+   satisfy the number of requested GPU slots.
+
+-  ``gres_supported``:
+      -  For Slurm, it indicates that ``GresTypes=gpu`` is set in the Slurm configuration, and nodes
+         with GPUs have properly configured GRES indicating the presence of any GPUs. The default is
+         true. When false, Determined will request ``slots_per_trial`` nodes and utilize only GPU 0
+         on each node unless slurm.slots_per_node is specified. It is the user’s responsibility to
+         ensure that GPUs will be available on nodes selected for the job using other configurations
+         such as targeting a specific resource pool with only GPU nodes or specifying a Slurm
+         constraint in the experiment configuration.
+
+      -  For PBS, the ngpus resource can be used to identify the number of GPUs available on a node.
+
+-  ``default_aux_resource_pool``: The default resource pool to use for tasks that do not need
+   dedicated compute resources, auxiliary, or systems tasks. Defaults to the Slurm/PBS default
+   partition if no resource pool is specified.
+
+-  ``default_compute_resource_pool``: The default resource pool to use for tasks that require
+   compute resources, e.g. GPUs or dedicated CPUs. Defaults to the Slurm/PBS default partition if it
+   has GPU resources and if no resource pool is specified.
+
+-  ``job_project_source``: Identifies the source to be used when generating a Slurm Workload
+   Characterization Key (WCKey), or PBS project name.
+
+Experiment Configuration
+========================
+
+The experiment contributes the following resource scheduling configuration. See the
+:ref:`experiment-config-reference` for the full details for these configuration options.
+
+-  ``resource_pool``: Identifies the queue/partition to be used.
+
+-  ``slots_per_trial``: The number of slots to use for each trial of this experiment. The default
+   value is 1; specifying a value greater than 1 means that multiple GPUs will be used in parallel.
+   Training on multiple GPUs is done using data parallelism.
+
+-  ``slurm``:
+      -  ``sbatch_args``: Array of Slurm options added as ``#SBATCH`` options in the generated batch
+         script
+      -  ``slots_per_node``: The minimum number of slots required for a node to be scheduled during
+         a trial. It must be greater than slots_per_trial. The default value is 1.
```

Comment:
Must be greater than slots_per_trial?  Shouldn't slots_per_trial be a multiple of slots_per_node?

Reply:
Dropped details.